backend/src/main/java/de/neuefische/teddyscoronadiaries/security/AppUser.java

```diff
@@ -0,0 +1,21 @@
+package de.neuefische.teddyscoronadiaries.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection = "appusers")
+public class AppUser {
+
+    @Id
+    private String googleId;
```

Comment:
i would just call it id, because it is your userId. If you also want to add a github login for example you could get a naming problem